calculus/integration-partial-fractions.md

```diff
@@ -1 +1,117 @@
 # Partial fraction decomposition
+## Shortcut: Heaviside Cover-up Method
+
+$$ 
```

Comment:
This detailed example should do inside an `{example}` directive to delineate it from the rest of the section. The example should also include an explanation of what is being solved for/what is being demonstrated. A short statement in the form "Given [blank], find [blank]" would suffice.

Make sure to do this for the rest of the examples.